jina/serve/runtimes/gateway/__init__.py

```diff
@@ -3,13 +3,17 @@
 from jina.serve.networking import GrpcConnectionPool
 from jina.serve.runtimes.asyncio import AsyncNewLoopRuntime
 from jina.serve.runtimes.gateway.graph.topology_graph import TopologyGraph
+from jina.serve.runtimes.mixin import MonitoringMixin
 
 
-class GatewayRuntime(AsyncNewLoopRuntime, ABC):
+class GatewayRuntime(AsyncNewLoopRuntime, MonitoringMixin, ABC):
     """
     The Runtime from which the GatewayRuntimes need to inherit
     """
 
+    def _setup_monitoring(self):
+        MonitoringMixin._setup_monitoring_server(self)
```

Comment:
u should use `self._setup_monitoring_server()` or why is this not added directly in the Mixin?

Reply:
The idea is that we could want to define some metrics on the gateway level and we can't do it the `MonitoringMixin` because it is shared for gateway and worker.